scanomatic/server/interface_builder.py

```diff
@@ -215,20 +215,22 @@ def _server_get_status(self, user_id=None):
         return sanitize_communication(_SOM_SERVER.get_server_status())
 
     def _server_get_scanner_status(self, user_id=None):
-        global _SOM_SERVER
         if _SOM_SERVER is None:
             self.logger.error("Server not fully initialized")
             return []
         if not _SOM_SERVER.scanner_manager.connected_to_scanners:
+            self.logger.error("Scanner manager not connected to scanner")
             return []
-        else:
-            return sanitize_communication(sorted(
-                [
-                    ScannerFactory.to_dict(scanner_model)
-                    for scanner_model in _SOM_SERVER.scanner_manager.status
-                ],
-                key=lambda x: x['socket'],
-            ))
+        self.logger.debug(
+            f"Scanner manager status: {_SOM_SERVER.scanner_manager.status}"
+        )
+        return sanitize_communication(sorted(
+            [
+                ScannerFactory.to_dict(scanner_model)
```

Comment:
this might be the problem, we should probably not use or have the `to_dict` left unless it does something very magical and instead use `jsonizer.dumps`

Reply:
There is some magic going on it seems. `jsonizer.dumps` causes scanners to not show up

Reply:
The `dumps` doesn't make an identical structure to `to_dict`, keeping track of types etc, so probably rather than magic, it is a matter of different expectations?

Reply:
ah yes this is for rpc communication and that shouldn't need preserializing everything. But now looks nice I think